src/Apps/Auction/Components/AuctionFAQ.tsx

```diff
@@ -0,0 +1,158 @@
+import { Box, media, Serif, space, Spacer, Theme, Toggle } from "@artsy/palette"
+import { AuctionFAQ_viewer } from "__generated__/AuctionFAQ_viewer.graphql"
+import { AuctionFAQQuery } from "__generated__/AuctionFAQQuery.graphql"
+import { useSystemContext } from "Artsy"
+import { renderWithLoadProgress } from "Artsy/Relay/renderWithLoadProgress"
+import React from "react"
+import Markdown from "react-markdown"
+import { createFragmentContainer, graphql, QueryRenderer } from "react-relay"
+import styled from "styled-components"
+
+interface AuctionFAQProps {
+  viewer: AuctionFAQ_viewer
+}
+
+export const AuctionFAQ: React.FC<AuctionFAQProps> = ({ viewer }) => {
+  return (
+    <Theme>
+      <AuctionWrapper>
+        <Serif size="8" color="black100">
+          Auction FAQs
+        </Serif>
+        <Spacer mb={2} />
+        <Serif size="3" color="black100">
+          How can we help you? Below are answers to some of the most common
+          questions from collectors.
+          <br />
+          Need more immediate assistance? Please{" "}
+          <a href="mailto:support@artsy.net">contact us</a>.
+        </Serif>
+        <Spacer mb={2} />
+        <Toggle label="Bidding">
+          <StyledMarkdown
+            source={viewer.bidding && viewer.bidding.content}
+            containerTagName="div"
+            unwrapDisallowed
+          />
+        </Toggle>
+        <Toggle label="Buyer's Premium, Taxes & Fees">
+          <StyledMarkdown
+            source={
+              viewer.buyersPremiumTaxesAndFees &&
+              viewer.buyersPremiumTaxesAndFees.content
+            }
+            containerTagName="div"
+            unwrapDisallowed
+          />
+        </Toggle>
+        <Toggle label="Payments and Shipping">
```

Comment:
If there's no `content` for these section, we probably shouldn't be showing the section headers either.